tests/nnstreamer_converter/unittest_converter.cc

```diff
@@ -0,0 +1,209 @@
+/**
+ * @file        unittest_converter.cc
+ * @date        18 Mar 2021
+ * @brief       Unit test for tensor_converter
+ * @see         https://github.com/nnstreamer/nnstreamer
+ * @author      Gichan Jang <gichan2.jang@samsung.com>
+ * @bug         No known bugs
+ */
+
+#include <gtest/gtest.h>
+#include <glib.h>
+#include <gst/gst.h>
+#include <tensor_common.h>
+#include <unittest_util.h>
+#include <tensor_converter_custom.h>
+#include <flatbuffers/flexbuffers.h>
+
+#define TEST_TIMEOUT_MS (1000U)
+
+static int data_received;
+/**
+ * @brief custom callback function
+ */
+GstBuffer * tensor_converter_custom_cb (GstBuffer *in_buf,
+    GstTensorsConfig *config) {
+  GstMemory *in_mem, *out_mem;
+  GstBuffer *out_buf = gst_buffer_new ();
+  GstMapInfo in_info;
+  guint mem_size;
+  gpointer mem_data;
+
+  if (!in_buf || !config)
+    return NULL;
+
+  data_received++;
+  in_mem = gst_buffer_peek_memory (in_buf, 0);
+  if (gst_memory_map (in_mem, &in_info, GST_MAP_READ) == FALSE) {
+    ml_loge ("Cannot map input memory / tensor_converter::flexbuf.\n");
+    return NULL;

```

Comment:
possible mem leak :
gst_buffer_unref (out_buf) before return, or create new buffer after this condition.

Reply:
Changed to create the buffer later. Thanks.